R/log.R

```diff
@@ -285,8 +288,39 @@ log_write <- function(file = NA,
                         write_log_element("log_path", "Log path: "))
 
    writeLines(cleaned_log_vec,
-              con = file.path(get_log_element("log_path"),
-                              get_log_element("log_name")))
+                 con = file.path(get_log_element("log_path"),
+                                 get_log_element("log_name")))
+   if (include_rds){
+      rds_fields <- c(
+         "end_time", "start_time", "run_time", "user", "hash_sum",
+         "log_path", "log_name", "file_path", "file_name",
+         "unapproved_packages_functions", "errors", "warnings"
+      )
+      log_options <- as.list(getOption('log.rx'))
+      cleaned_log_list <- Map(
```

Comment:
Not sure what @nicholas-masel or @bms63 but everywhere else we are using {tidyverse} map variants so I would say we should keep consistent and use them here,

Reply:
Thanks! Updated.